.circleci/config.yml

```diff
@@ -159,12 +159,7 @@ commands:
       - run:
             name: Install Mephisto
             command: |
-              cd ..
-              git clone git@github.com:facebookresearch/Mephisto.git Mephisto
-              cd Mephisto; git checkout v0.4.2 -b stable
-              pip install -r requirements.txt
-              python setup.py develop
-              # `echo` so that ENTER will be pressed at the prompt
+              pip install mephisto==1.0.0b2
```

Comment:
why b2?

Reply:
1.0.0 full isn't out until later today, but wanted to get started on this flow sooner.